bin/i18n/sync-up.rb

```diff
@@ -12,7 +12,18 @@ def sync_up
 
   CROWDIN_PROJECTS.each do |name, options|
     puts "Uploading source strings to #{name} project"
-    system "crowdin --config #{options[:config_file]} --identity #{options[:identity_file]} upload sources"
+    command = "crowdin --config #{options[:config_file]} --identity #{options[:identity_file]} upload sources"
+    Open3.popen2(command) do |_stdin, stdout, status_thread|
+      stdout.each_line do |line|
+        # skip lines detailing individual file upload, unless that file
+        # resulted in an unexpected response
+        next if line.start_with?("File ") && line.end_with?("OK\n", "SKIPPED\n")
+
+        puts line
+      end
+
+      raise "Sync up failed"  unless status_thread.value.success?
```

Comment:
* nit-pick: extra whitespace before "unless"
* Although I should have though of this in the previous PR, but would it also be helpful to log the value of `status_thread.value`?

Reply:
I don't think it's necessary to explicitly log the status; we communicate a failure status by explicitly raising and a success status by continuing to execute.